### PR TITLE
Glyph Interface

### PR DESCRIPTION
**Pull request by 25huizengek1**
This pull request tracks the progress of the Nothing Glyph Interface implementation.

**Currently WIP**, so just a draft.

Relevant discussion: https://github.com/25huizengek1/ViTune/discussions/192

### CODE REVIEW

File: app/vendor/KetchumSDK_Community_20240307.jar
Comment (dtomvan) [inline app/vendor/KetchumSDK_Community_20240307.jar:1]: Is this strictly needed in JAR form? Is this some official sdk from Nothing we don't have the source to, or should we use a submodule?

**Review by dtomvan — COMMENTED** [review]
Code LGTM. Although you understand this better than I do.

**Comment by dtomvan** [comment]
Also, in the future we could get a barebones visualization, since we are already normalizing we should be able to easily see peaks in the audio, so we could have some other glyph "light up on the beat" or something, like those generic "Trap City" visualizations on YouTube...

File: app/vendor/KetchumSDK_Community_20240307.jar
Comment (dtomvan) [reply app/vendor/KetchumSDK_Community_20240307.jar:1]: Alright, seems clean: https://www.virustotal.com/gui/file/263572f2a044f10793e179ecb16fb7ccdda5196f8dd0c789552a4e6d9702b83b?nocache=1

Comment (25huizengek1) [reply app/vendor/KetchumSDK_Community_20240307.jar:1]: There is no source for the Nothing SDK available, so I think manual dependency management is a good idea.

Alternatively I could try to reverse-engineer Nothing's system service in order to do the exact same thing, but I guess this works fine.

**Comment by 25huizengek1** [comment]
> Also, in the future we could get a barebones visualization, since we are already normalizing we should be able to easily see peaks in the audio, so we could have some other glyph "light up on the beat" or something, like those generic "Trap City" visualizations on YouTube...

Music visualization is already supported by Nothing.

**Comment by dtomvan** [comment]
Merged it 👍🏼

**Comment by 25huizengek1** [comment]
@dtomvan Should merge after Nothing GDK Team's approval. Still ongoing.

**Pull request merged**